### DISCUSSION TITLE
Do you have a more detailed tables tutorial

### DISCUSSION THREAD

**Discussion by cn-hew** [fltk-rs/fltk-rs#975] [Q&A]
I'm already able to run the examples. I want to use fltk RS more deeply, but I don't have experience in developing GUI. I know what I want to do, but I don't know how to start

Can you give some tutorials related to tables?

**Comment by MoAlyousef** [comment]
Hi
Currently there are video and written tutorials:
- https://github.com/fltk-rs/fltk-rs/wiki
- https://youtube.com/c/MoAlyousef

There aren’t tutorials specifically for tables, but I’ll hopefully create one this weekend and post it to my youtube channel. 
I also plan to create a new fltk-table crate which should make creating tables easier by reducing the required boilerplate.

**Comment by cn-hew** [comment]
Thank you for your help
I made a mistake when using `set_value`. Is it my mistake?
![image](https://user-images.githubusercontent.com/31717413/140030183-7c5d6442-e120-4bab-b7a1-f7ab222b048c.png)

**Comment by MoAlyousef** [comment]
set_value() has no return value, so yes that’s a mistake. Not all widget methods return `self`. 
You’ll have to construct input, then call `input.set_value()` separately.

**Reply by cn-hew** [reply]
I have used the methods in your video tutorial and have been able to achieve my goal

**Comment by MoAlyousef** [comment]
I have created an fltk-table [crate](https://crates.io/crates/fltk-table) and uploaded a new [tutorial](https://www.youtube.com/watch?v=pVJ8Yq1kDGs) on using it. It should simplify creating tables by a lot.

**Reply by cn-hew** [reply]
thank you for help,you replied so quickly
